### PR TITLE
Fixes & Updates for "Components in a Block" section

### PR DESCRIPTION
**Pull request by piqusy**
# Description

Applied for latest release ATM

1. Fix broken links
2. Remove docs for non-existing PHP helpers
3. Update content
4. Add anchor link for a reference
5. Remove unnecessary fragments for an example

### CODE REVIEW

**Review by iruzevic — APPROVED** [review]

File: website/docs/basics/blocks-component-in-block.md [hunk 1]
```diff
@@ -590,6 +590,8 @@ export const HeadingEditor = ({ attributes, setAttributes }) => {
 
 In JavaScript, you spread the results of the props helper.
 
+And this is how the PHP template looks like.
```
Comment (goranalkovic-infinum) [inline website/docs/basics/blocks-component-in-block.md:593]:
```suggestion
The PHP template should look like this:
```

File: website/docs/basics/blocks-component-in-block.md [hunk 2]
```diff
@@ -728,7 +728,7 @@ You can also do this and this is why we created the third parameter in the props
 
 typographyContent attribute will follow the naming prefix when passed to a child but the disableEdit will remain the same name in all the children.
 
-Here, as a third parameter, you can also pass all include attributes but they will not get prefixed:
+Any of the included attributes can be passed as a third parameter here, but they will not be prefixed:
```
Comment (goranalkovic-infinum) [inline website/docs/basics/blocks-component-in-block.md:731]:
```suggestion
Any of the included attributes can be passed as the third parameter here, but they will not be prefixed:
```

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]
Minor suggestions 😄

**Review by iobrado — APPROVED** [review]

File: website/docs/basics/blocks-component-in-block.md (hunk 2 above)
Comment (piqusy) [reply website/docs/basics/blocks-component-in-block.md:731]: Will not be prefixed?

**Review by volfkarlo — APPROVED** [review]

File: website/docs/basics/blocks-component-in-block.md (hunk 2 above)
Comment (piqusy) [reply website/docs/basics/blocks-component-in-block.md:731]: Oh sorry, I misread that... ignore my comment here.

**Review by goranalkovic-infinum — APPROVED** [review]

**Pull request merged**